test/e2e/framework/util/job/util.go

```diff
@@ -0,0 +1,25 @@
+package job
+
+import (
+	batch "k8s.io/api/batch/v1"
+	batchv1 "k8s.io/api/batch/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func newBool(val bool) *bool {
```

Comment:
Seems like we can just move this to the same file as `NewTestJob` and just remove this file entirely

Reply:
I found out that the newBool was only called in newTestJob, so I manually initialize a bool value and remove the newBool function from the pkg.